### PR TITLE
ascanrules (alpha/beta/sqli): Alert text fixes (periods and whitespace(s))

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
- CHANGELOG > Added fix note.
- Messages.properties > Updated various resource messages, removed some that were unused.

---

I also tackled the Adv SQLi add-on as part of this PR, it only needed the addition of a single period and already has a maint note in its CHANGELOG.

## Related Issues
zaproxy/zap-extensions#5522

## Checklist
- [na] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [x] Write tests
- [x] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties:1]: The following should have a dot too?
- `ascanbeta.HTTPParamPoll.sol` 
- `ascanbeta.backupfiledisclosure.desc`
- `ascanbeta.cookieslack.desc`
- `ascanbeta.httpoxy.desc`
- `ascanbeta.sessionfixation.soln`
- `ascanbeta.sessionidaccessiblebyjavascript.alert.extrainfo`
- `ascanbeta.shellshock.soln`
- `ascanbeta.sourcecodedisclosure.gitbased.soln`
- `ascanbeta.sourcecodedisclosure.svnbased.soln`

Double dot in `ascanbeta.insecurehttpmethod.put.exploitable.desc`.

File: addOns/sqliplugin/src/main/resources/org/zaproxy/zap/extension/sqliplugin/resources/Messages.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sqliplugin.alert.desc = A SQL injection may be possible using the attached payload
+sqliplugin.alert.desc = A SQL injection may be possible using the attached payload,
```
Comment (thc202) [inline addOns/sqliplugin/src/main/resources/org/zaproxy/zap/extension/sqliplugin/resources/Messages.properties:1]: Was this meant to be a comma?

Comment (kingthorin) [reply addOns/sqliplugin/src/main/resources/org/zaproxy/zap/extension/sqliplugin/resources/Messages.properties:1]: Nope, oops

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**